styleguide/STYLEGUIDE.md

```diff
@@ -1,26 +1,23 @@
-## General
-* We care about clean code and aim to make this codebase as self-documenting and readable as possible.
-* We primarily use Kotlin and follow the standard coding conventions outlined in https://kotlinlang.org/docs/reference/coding-conventions.html
+## DuckDuckGo Android Style Guide
+* It is possible to get Android Studio to adhere to these style guidelines, via a rather complex sequence of menus. To make it easier, we've provided a coding style that can be imported into Android Studio.
```

Comment:
Why has "We care about clean code and aim to make this codebase as self-documenting and readable as possible." been removed? accidentally?

This seems to be duplicate instructions of the Android Studio Setup